core/src/main/scala/com/clairvoyant/restonomer/core/app/RestonomerContext.scala

```diff
@@ -30,21 +30,34 @@ class RestonomerContext(val restonomerContextDirectoryPath: String) {
 
   val configVariables: Map[String, String] = loadConfigVariables(CONFIG_VARIABLES_FILE_PATH)
 
-  val checkpoints: List[CheckpointConfig] = loadConfigsFromDirectory[CheckpointConfig](
-    configDirectoryPath = CHECKPOINTS_CONFIG_DIRECTORY_PATH,
-    configVariablesSubstitutor = ConfigVariablesSubstitutor(configVariables = configVariables)
-  )
-
-  def runCheckpoint(checkpointName: String): Unit =
-    checkpoints
-      .find(_.name == checkpointName) match {
-      case Some(checkpointConfig) =>
-        runCheckpoint(checkpointConfig)
-      case None =>
-        throw new RestonomerContextException(s"The checkpoint: $checkpointName does not exists.")
+  def runCheckpoint(checkpointName: String): Unit = {
+    val checkpoints: List[CheckpointConfig] = loadConfigsFromDirectory[CheckpointConfig](
+      configDirectoryPath = CHECKPOINTS_CONFIG_DIRECTORY_PATH,
+      configVariablesSubstitutor = ConfigVariablesSubstitutor(configVariables = configVariables),
+      fileName = Option(checkpointName)
+    )
+    checkpoints.foreach { checkpointConfig =>
+      println(s"Checkpoint Name -> ${checkpointConfig.name}\n")
+      runCheckpoint(checkpointConfig)
+      println("\n=====================================================\n")
     }
+  }
+
+  def runCheckpointWithPath(checkpointPath: String): Unit = {
+    val filePath = s"$restonomerContextDirectoryPath/$checkpointPath"
```

Comment:
This need to be changed so that user can provide path relative to checkpoints directory and not to context directory.

Reply:
done